homeassistant/components/knocki/manifest.json

```diff
@@ -4,7 +4,7 @@
   "codeowners": ["@joostlek", "@jgatto1", "@JakeBosh"],
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/knocki",
-  "integration_type": "device",
+  "integration_type": "service",
```

Comment:
If physical devices are involved at all, we don't label it as a service. `hub` seems appropriate.

https://developers.home-assistant.io/docs/creating_integration_manifest#integration-type